test/Project.toml

```
@@ -16,7 +16,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 CSV = "0.10"
-CairoMakie = "0.12"
+CairoMakie = "0.12, 0.13"
```

```suggestion
CairoMakie = "0.13"
```
Is there any reason for keeping the old compat?